src/components/MerchantPayout/VoucherStatusModal/VoucherStatusModal.tsx

```diff
@@ -132,5 +145,5 @@ export const VoucherStatusModal: FunctionComponent<VoucherStatusModal> = ({
       <View style={styles.background} />
       <View style={styles.cardWrapper}>{card}</View>
     </Modal>
-  );
+  ) : null;
```

Comment:
noticed an issue when manually keying in voucher codes, `ManualAddVoucherModal` cannot be closed or used to add any other codes